tests/integration/targets/aci_domain_to_encap_pool/tasks/main.yml

```diff
@@ -89,6 +92,9 @@
     vm_provider: vmware
     state: present
   register: vmm_domain
+  when:
```

Comment:
Does this fails on 3.2? This conditions are blocking testing on 3.2

Reply:
yes it fails on 3.2 hence the condition